common/src/main/kotlin/flank.common/Os.kt

```diff
@@ -0,0 +1,13 @@
+package flank.common
+
+private val osName = System.getProperty("os.name")?.toLowerCase() ?: ""
+
+val isMacOS: Boolean by lazy {
```

Comment:
Do we need a isLinux? or is it the "else" case?

Reply:
basically we just use `isWindows` and `isMacOS`